hollow/src/main/java/com/netflix/hollow/api/metrics/HollowConsumerMetrics.java

```diff
@@ -18,9 +18,31 @@
 
 import com.netflix.hollow.core.read.engine.HollowReadStateEngine;
 
+import java.util.concurrent.atomic.AtomicLong;
+
 public class HollowConsumerMetrics extends HollowMetrics {
     private int refreshFailed;      // TODO: Move these metrics over to com.netflix.hollow.api.consumer.metrics.AbstractRefreshMetricsListener
     private int refreshSucceeded;
+    private AtomicLong lastRefreshStartTimeNs; //last time a refresh started
```

Comment:
We cannot move this to the listener because it is not invoked until [later in the update](https://github.com/Netflix/hollow/blob/master/hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java#L142-L144), and also because of the sequence of the listener, it cannot record both start and end time at the very beginning and end. The value will be read from another thread to report metrics, hence we need it to be threadsafe.